### PR TITLE
[SRVCOM-2105] Serverless: s/v1alpha/v1beta1 in examples

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/SRVCOM-2105

Link to docs preview:
Same string substitution across many files, no need to look at the preview, source review is enough.

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/serverless-kafka-sasl-channels.adoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ $ oc edit knativekafka
 +
 [source,yaml]
 ----
-apiVersion: operator.serverless.openshift.io/v1alpha1
+apiVersion: operator.serverless.openshift.io/v1beta1
```
Comment (skonto) [inline modules/serverless-kafka-sasl-channels.adoc:56]: All kafka ones stay as v1alpha1. Only Eventing & Serving are moving.

File: modules/serverless-kafka-sink.adoc [hunk 2]
```diff
@@ -22,7 +22,7 @@ You can create an event sink called a Kafka sink that sends events to a Kafka to
 .Kafka sink YAML
 [source,yaml]
 ----
-apiVersion: eventing.knative.dev/v1alpha1
+apiVersion: eventing.knative.dev/v1beta1
```
Comment (skonto) [inline modules/serverless-kafka-sink.adoc:25]: Only the operator ones with prefix  ```operator.serverless.openshift.io/``` for Serving & Eventing have changed. cc @pierDipi

File: modules/serverless-configuring-event-delivery-examples.adoc [hunk 3]
```diff
@@ -19,7 +19,7 @@ spec:
   delivery:
     deadLetterSink:
       ref:
-        apiVersion: eventing.knative.dev/v1alpha1
+        apiVersion: eventing.knative.dev/v1beta1
```
Comment (mgencur) [inline modules/serverless-configuring-event-delivery-examples.adoc:22]: Only operator.knative.dev (KnativeServing,KnativeEventing) should be changed. Not `eventing.knative.dev`

**Review by mgencur — CHANGES_REQUESTED** [review]
Hey @rh-max , I've stopped commenting.  We should be changing only group `operator.knative.dev`  to v1beta1. No other groups. There are many places in this PR where other groups are changed.

**Comment by mgencur** [comment]
LGTM. I'm not able to tell if all occurrences were changed but I trust you :)

**Comment by rh-max** [comment]
Hey @skonto and @mgencur , thanks.

Stavros, I implemented Martin's version, so the change to `v1beta1` is now only in `operator.knative.dev` examples. You wrote something different (`operator.serverless.openshift.io/`), let me know if you disagree with the current version.

**Review by lahinson — COMMENTED** [review]
lgtm

**Review by maxwelldb — APPROVED** [review]
Spot checked in preview, e.g. https://58245--docspreview.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-configuration.html.

**Pull request merged**